internal/cmd/local/command_kafka_start.go

```diff
@@ -127,46 +127,109 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 	}
 	ports := c.Config.LocalPorts
 	platform := &specsv1.Platform{OS: "linux", Architecture: runtime.GOARCH}
-	natKafkaRestPort := nat.Port(ports.KafkaRestPort + "/tcp")
-	natPlaintextPort := nat.Port(ports.PlaintextPort + "/tcp")
-	config := &container.Config{
+	natKafkaRestPorts := []nat.Port{nat.Port(ports.KafkaRestPorts[0] + "/tcp"), nat.Port(ports.KafkaRestPorts[1] + "/tcp")}
+	natPlaintextPorts := []nat.Port{nat.Port(ports.PlaintextPorts[0] + "/tcp"), nat.Port(ports.PlaintextPorts[1] + "/tcp")}
+	containerStartCmd := strslice.StrSlice{"bash", "-c", "'/etc/confluent/docker/run'"}
+
+	// create a customized network
+	_, err = dockerClient.NetworkCreate(
+		context.Background(),
+		"confluent-local-network",
+		types.NetworkCreate{
+			CheckDuplicate: true,
+			Driver:         "bridge",
+		},
+	)
+	if err != nil && !strings.Contains(err.Error(), "already exists") {
+		return err
+	}
+
+	config1 := &container.Config{
 		Image:    dockerImageName,
-		Hostname: "broker",
-		Cmd:      strslice.StrSlice{"bash", "-c", "'/etc/confluent/docker/run'"},
+		Hostname: "broker1",
```

Comment:
suggest hyphenating `broker-<ID>`.  reads a little more clearly. might be more consistent with other examples too -- I doubt we're consistent everywhere, but a couple of public multibroker examples that I checked do hyphenate (MRC [example](https://github.com/confluentinc/examples/blob/7.4.0-post/multiregion/docker-compose.yml#L67), kafka-images KRaft [example](https://github.com/confluentinc/kafka-images/blob/master/examples/confluent-server-kraft/docker-compose.yml#L20))

Reply:
agree. This is also being used as the name of the containers too, so I'm thinking something more explicit like `confluent-local-broker-1`